src/test/groovy/de/dkfz/roddy/execution/jobs/cluster/lsf/LSFJobManagerSpec.groovy

```diff
@@ -181,6 +183,29 @@ class LSFJobManagerSpec extends Specification {
         jobInfo.jobID.toString() == "22005"
     }
 
+    String dateToString(ZonedDateTime date) {
+        DateTimeFormatter.ofPattern('MMM ppd HH:mm').format(date)
+    }
+
+    void "test parseTime"() {
+        given:
+        JobManagerOptions parms = JobManagerOptions.create().build()
```

Comment:
I am currently working on right this topic. I will take a look at your tests and use what I can in the new helper class and test class.